tests/Makefile

```diff
@@ -241,17 +241,20 @@ run-listing-fuzzer:
 		--rm \
 		--restart=Never \
 		--attach=True \
-		--image=zenko/s3fuzz:latest \
+		--image=tmacro/s3fuzz:latest \
```

Comment:
Do we want to pull from your personal docker hub image?

Reply:
That is of concern, we should be using zenko's registry

Reply:
`custom docker image left in purposefully until done with testing`

Reply:
Would it be possible to push a custom test image to the zenko account in docker hub instead? If we do that, the test won't be bound to your account.

Reply:
I meant the second, I'm doing some testing to make sure its deterministic and then I'll swap it over